<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>743</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Жёсткая система, Рунге-Кутта 2 (16.61), Ноздрин</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>15</number>
    </property>
    <item row="1" column="0">
     <widget class="QTableWidget" name="table">
      <property name="maximumSize">
       <size>
        <width>800</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerPixel</enum>
      </property>
      <column>
       <property name="text">
        <string>x</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>u1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>u2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>v1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>v2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>E1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>E2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ex(-0.01x)</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ex(-1000x)</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="system_label">
      <property name="maximumSize">
       <size>
        <width>800</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="2" rowspan="3">
     <widget class="Mpl_widget" name="plot" native="true">
      <property name="minimumSize">
       <size>
        <width>640</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="3">
        <widget class="QLabel" name="label_3">
         <property name="maximumSize">
          <size>
           <width>23</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;ep&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:7pt; font-weight:600;&quot;&gt;Шаг&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="2" column="7">
        <widget class="QLineEdit" name="eps_text_box">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>0.0001</string>
         </property>
         <property name="placeholderText">
          <string>0.0001</string>
         </property>
        </widget>
       </item>
       <item row="3" column="7">
        <widget class="QLineEdit" name="x_max_text_box">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>15</string>
         </property>
         <property name="placeholderText">
          <string>10</string>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLabel" name="label_2">
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:9pt; font-weight:600;&quot;&gt;x&lt;/span&gt;&lt;span style=&quot; font-size:9pt; font-weight:600; vertical-align:sub;&quot;&gt;max&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="step_control_check_box">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Контроль</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="plot_btn">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Запуск</string>
         </property>
        </widget>
       </item>
       <item row="0" column="7">
        <widget class="QLineEdit" name="step_text_box">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>0.001</string>
         </property>
         <property name="placeholderText">
          <string>0.001</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Mpl_widget</class>
   <extends>QWidget</extends>
   <header>mpl_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
